Frontend/src/components/ProfileImage/index.jsx

```diff
@@ -1,18 +1,68 @@
-import React from 'react';
-import profileDefault from 'assets/user.png';
+import React, { useState, useEffect } from 'react';
+import Avatar from '@mui/material/Avatar';
 
+import ProfileImageUploadDialog from 'components/general/upload/ProfileImageUploadDialog';
 import styles from './profileImage.module.css';
 
-function ProfileImage({ sizePixels }) {
+import stringToColor from 'shared/utils/stringColor';
+import getProfile from 'shared/services/profile';
+
+function ProfileImage({ sizePixels, editable = false, overrideProfileImage = null }) {
+  const [user, setUser] = useState({});
+  async function loadUser() {
+    setUser(await getProfile());
+  }
+  useEffect(loadUser, [setUser]);
```

Comment:
is this working? shouldn't it be like `useEffect(()=>{ loadUser()} ,[])`

Reply:
It is, I mean in a functional programming language like JS `()=>{ loadUser()}` and just `loadUser` are (almost) equivalent except for parameters, which there are none here, right? I think it is possible to leave out the `[setUser]` part, but it shouldn't hurt 🤔